src/pages/installing/using-image-mirroring.md

```diff
@@ -6,14 +6,15 @@ slug: using-image-mirroring
 toc: true
 ---
 
-Setting up container registry repository mirroring enables you to do the following:
+Image mirroring is a technique to pull a container image from one or more registries using the same fully qualified image name.  Setting up container registry repository mirroring enables you to do the following:
 
 - Configure your OpenShift Container Platform cluster to redirect requests to pull images from a repository on a source image registry and have it resolved by a repository on a mirrored image registry.
 - Identify multiple mirrored repositories for each target repository, to make sure that if one mirror is down, another can be used.
+- Enable IBM ACD Container Edition to pull images from the new Azure ACD image registry so IBM ACD will continue to run after December 31, 2022.
```

Comment:
from the new Merative ACD image registry (vs Azure )

Reply:
Fixed